fem/fespace.cpp

```diff
@@ -2160,6 +2168,22 @@ void FiniteElementSpace::Destroy()
       delete [] bdofs;
       delete [] fdofs;
    }
+#ifdef MFEM_USE_CEED
+   for(auto entry : internal::ceed_basis_map)
+   {
+      if (std::get<0>(entry.first)==this)
+      {
+         internal::ceed_basis_map.erase(entry.first);
```

Comment:
This would need to call the libCEED destructors to not leak, right?

Reply:
Like this: https://github.com/mfem/mfem/blob/a40523a7cfe6bb0037311cd46200e2a5eddfc180/general/device.cpp#L164

Reply:
I think it's good now.